bip-0039/indonesian.txt

```diff
@@ -0,0 +1,2048 @@
+rusak
+rute
+rutin
+saat
+sabda
+sabuk
+sadar
+saham
+sains
+saja
+sakit
+saksi
+sakti
+salah
+saleh
+saling
+salju
+saluran
+sama
+sambil
+sampai
+samurai
+sana
+sandiwara
+sangat
+sanksi
+santri
+sapi
+sarana
+sari
+sarjana
+sasaran
+sastra
+satelit
+satu
+satwa
+saudara
+saus
+sawah
+sawit
+saya
+sayuran
+seakan
+sebagai
+sebesar
+sebuah
+secara
+sedang
+sederhana
+sedikit
+seekor
+segala
+segera
+segi
+segmen
+sehari
+sehingga
+sehubungan
+seimbang
+seiring
+sejak
+sejenis
+sejumlah
+sekarang
+sekelompok
+sekitar
+sekolah
+sekretaris
+seksual
+sektor
+sekutu
+selama
+selesai
+selisih
+seluruh
+semakin
+sembilan
+sementara
+seminggu
+semoga
+sempat
+semua
+senang
+sendiri
+sengaja
+seni
+senjata
+sensus
+sentral
+senyawa
+seolah
+seorang
+sepanjang
+seperti
+sepuluh
+serangan
+serba
+serentak
+sering
+serta
+serupa
+server
+sesama
+seseorang
+sesi
+sesuatu
+setahun
+setelah
+setiap
+setuju
+seumur
+sewaktu
+siang
+siapa
+siaran
+sibuk
+sidang
+sifat
+signifikan
+sihir
+sikap
+siklus
+silat
+silinder
+silsilah
+siluman
+simbol
+sinar
+sinetron
+singel
+sini
+sintesis
+sinyal
+sipil
+sirip
+sirkuit
+sisa
+sisi
+sistem
+siswa
+situs
+skala
+skandal
+skema
+skenario
+sketsa
+skor
+soal
+solusi
+sopan
+sore
+sosial
+sosok
+spektrum
+spesies
+spiritual
+spons
+stabil
+stadion
+staf
+standar
+star
+stasiun
+status
+stop
+strategis
+string
+stroke
+struktur
+studi
+suaminya
+suara
+suasana
+suatu
+subjek
+subspesies
+subur
+subyek
+suci
+sudah
+sudut
+suhu
+suka
+sukses
+suku
+sulit
+sultan
+sulung
+sumber
+sumpah
+sumur
+sungai
+suntingan
+supaya
+support
+surat
+surel
+surga
+surut
+survei
+surya
+susah
+susu
+sutradara
+swasta
+syair
+syarat
+syukur
+syuting
+taat
+tabel
+tablet
+tabung
+tadi
+tahap
+tahta
+tahun
+tajam
+takhta
+taksi
+taktik
+takut
+tali
+taman
+tambahan
+tampil
+tamu
+tanah
+tanda
+tanggal
+tani
+tanjung
+tanpa
+tantangan
+tanya
+tapi
+tari
+tata
+tatkala
+tawaran
+tayang
+teater
+tebal
+tebing
+tebu
+tegas
+teguh
+tekanan
+teknologi
+teks
+telah
+televisi
+telinga
+telur
+teman
+tembok
+tempat
+temurun
+tenaga
+tendangan
+tengah
+tentang
+teologi
+teori
+tepat
+tepi
+tepung
+terakhir
+terbentuk
+tercatat
+terdapat
+terendah
+tergolong
+terhadap
+terima
+terjadi
+terkenal
+terletak
+termasuk
+ternyata
+terowongan
+terpilih
+tersebut
+tertentu
+terutama
+terwujud
+tetapi
+tewas
+tiada
+tiang
+tiap
+tiba
+tidak
+tidur
+tiga
+tiket
+tikus
+timah
+timbul
+timnya
+timur
+tindakan
+tinggi
+tinju
+tipe
+tipis
+tiri
+titik
+tokoh
+toleransi
+tolong
+tombol
+tongkat
+topeng
+topik
+tradisional
+tragedi
+trailer
+transportasi
+trek
+triliun
+tropis
+truk
+tsunami
+tuan
+tubuh
+tuduhan
+tugas
+tugu
+tujuan
+tukang
+tulang
+tulisan
+tumbuh
+tunai
+tunduk
+tunggal
+tuntutan
+turbin
+turis
+turnamen
+turut
+tutup
+uang
+ubah
+ubur
+ucapan
+udang
+udara
+ujian
+ujung
+ukiran
+ukuran
+ulama
+ulang
+ular
+ulasan
+umat
+umpan
+umumnya
+umur
+undang
+unggulan
+ungkapan
+ungu
+unik
+universitas
+unjuk
+unsur
+untuk
+upacara
+upah
+upaya
+upeti
```

Comment:
Sorry, by "this word" which one are you referring to? Because GitHub Web UI is showing 4 words: upacara, upah, upaya, upeti. I would argue that they are familiar to native speakers. What is your basis to say that they are not? Let's take "upeti" for example, this word is found 792 times in the Wikipedia article (see PR description which I have just updated to describe the process of producing the wordlist).

Reply:
> Native indonesian speaker here. Instead of the Wikipedia Corpus (which contain too many technical terminology), have we considered the "Sari Kata Bahasa Indonesia" book?
> 
> This book is used nationwide in junior/primary schools. It would ensure anyone with a basic education would be able to understand our wordlist.
> 
> This same book is also used as a reference to test if a foreigner can be permitted to work in Indonesian companies.

This is a good idea, but what about license? And is there an online version for it?